src/SWP/Bundle/ContentBundle/Resources/config/doctrine-phpcr/ArticleMedia.phpcr.yml

```diff
@@ -0,0 +1,32 @@
+SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\ArticleMedia:
+    referenceable: true
+    parentdocument: parent
+    nodename: id
+    fields:
+        body:
+            type: string
+        description:
+            type: string
+        located:
+            type: string
+        byLine:
+            type: string
+        mimetype:
+            type: string
+        usageTerms:
+            type: string
+    referrers:
+        renditions:
+            referringDocument: SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\ImageRendition
+            referencedBy: media
+            cascade: [remove]
+    referenceOne:
+        image:
+            targetDocument: SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\Image
```

Comment:
sometimes you provide class namespace in mapping files with leading `\` and sometimes not - let's make it consistent and do it in one way